js/index.js

```diff
@@ -0,0 +1,86 @@
+let startBtn = document.querySelectorAll(".btn--play");
+let pauseBtn = document.querySelectorAll(".btn--pause");
+let stopBtn = document.querySelectorAll(".btn--stop");
+let videos = document.querySelectorAll(".e__card__video");
+
+// Hide menu on mobile view after clicking on menu items
+function hideMenu() {
+  let menuOpen = document.querySelector("#h__side-menu").checked;
+
+  if ((menuOpen = true)) {
+    document.querySelector("#h__side-menu").checked = false;
+  }
+}
+
+// Functionalities of play/pause/stop buttons
+function videoControls() {
+  // Looping trough all the start buttons and applying the click event
+  for (let i = 0; i < videos.length; i++) {
+    startBtn[i].addEventListener("click", function () {
+      videos[i].contentWindow.postMessage(
+        '{"event":"command","func":"' + "playVideo" + '","args":""}',
+        "*"
+      );
+      timer();
+    });
+  }
+
+  // Looping trough all the pause buttons and applying the click event
+  for (let i = 0; i < videos.length; i++) {
+    stopBtn[i].addEventListener("click", function () {
+      videos[i].contentWindow.postMessage(
+        '{"event":"command","func":"' + "stopVideo" + '","args":""}',
+        "*"
+      );
+    });
+  }
+
+  // Looping trough all the stop buttons and applying the click event
+  for (let i = 0; i < videos.length; i++) {
+    pauseBtn[i].addEventListener("click", function () {
+      videos[i].contentWindow.postMessage(
+        '{"event":"command","func":"' + "pauseVideo" + '","args":""}',
+        "*"
+      );
+    });
+  }
+}
```

Comment:
will there be multiple videos on the same page?

Reply:
Yes we created a randomise effect, one video is shown at a time from an array of videos